.github/workflows/ci.yml

```diff
@@ -23,7 +23,8 @@ jobs:
         uses: actions/cache/@v1
         with:
           path: .cache/cypress
-          key: ${{ runner.os }}-cypress-cache-version-${{ steps.cypress-version.outputs.version }}
+          key:
```

Comment:
@NicholasBoll I was getting a cache error from multiple versions of `Linux-cypress-cache-version` being output here: https://github.com/Workday/canvas-kit/runs/1419196261.

Not sure why this happened all of a sudden, but adding `[0]` seems to have fixed it. Any ideas why this came up? It seems like this caching approach is deprecated, so maybe we should update it?

Reply:
`steps.cypress-version.outputs.version` contains the steps from the step with an id of `cypress-version` which is the output of `node utils/get-cypress-versions.js` which is going through the `yarn.lock` file and getting all versions of `cypress` found.

Cypress v5 is probably coming from the `cypress@*` requirement of `@testing-library/cypress`. So some dependency update forced an upgrade of Cypress to version 5. You could probably verify with a `yarn why cypress`. You could upgrade to `actions/cache/@v2`, but this problem would remain.

Reply:
2 versions of Cypress causes issues - we probably only want a single version of Cypress included in the `yarn.lock` file